legal-api/src/legal_api/config.py

```diff
@@ -79,6 +81,11 @@ class _Config:  # pylint: disable=too-few-public-methods
 
     COLIN_URL = f"{os.getenv('COLIN_API_URL', '')}{os.getenv('COLIN_API_VERSION', '')}"
     LEGAL_API_BASE_URL = f'{BUSINESS_API_URL + BUSINESS_API_VERSION_2}/businesses'
+
+    # This is to use reverse proxy to make filingLink, commentLink, and documentLink work
+    # This can be removed once the Migration to GCP is complete
```

Comment:
To be precise, this can be changed to using the BUSINESS API GW URL when the migration to GCP is complete.